packages/backend/src/jwt/types.ts

```diff
@@ -7,3 +9,15 @@ export type JwtReturnType<R, E extends Error> =
       data?: undefined;
       errors: [E];
     };
+
+export type MachineTokenReturnType<R, E extends Error> =
+  | {
+      data: R;
+      tokenType: NonSessionTokenType;
+      errors?: undefined;
+    }
+  | {
+      data?: undefined;
+      tokenType: NonSessionTokenType;
```

Comment:
This feels like weird naming haha

Reply:
haha it's awkward yeah 😆 

`MachineTokenType` sounds more direct